.github/workflows/ci.yml

```diff
@@ -21,9 +21,9 @@ on:
   schedule:
     - cron: '28 0 * * *'
   push:
-    branches: ['master', 'v1-10-test', 'v1-10-stable']
+    branches: ['master', 'v1-10-test', 'v1-10-stable', 'v2-0-test']
   pull_request:
-    branches: ['master', 'v1-10-test', 'v1-10-stable']
+    branches: ['master', 'v1-10-test', 'v1-10-stable', 'v2-0-test']
```

Comment:
Are we going to have PRs targeting v2-0-test now? I believe we only want to rebase v2-0-test periodically, but all the PRs should still go to master? In this case we do not need that - that woudl be harmful in fact because what do we do if someone starts making PR to v2-0-test and it is already behind master?